src/common/hotbar-store.ts

```diff
@@ -92,30 +92,13 @@ export class HotbarStore extends BaseStore<HotbarStoreModel> {
     return this.hotbarIndex(this.activeHotbarId);
   }
 
-  get defaultHotbarInitialItems() {
-    const { metadata: { uid, name, source } } = catalogEntity;
-    const initialItem = { entity: { uid, name, source }};
-
-    return [
-      initialItem,
-      ...Array.from(Array(defaultHotbarCells - 1).fill(null))
-    ];
-  }
-
-  get initialItems() {
+  static getInitialItems() {
     return [...Array.from(Array(defaultHotbarCells).fill(null))];
   }
 
-  @action protected async fromStore(data: Partial<HotbarStoreModel> = {}) {
-    if (data.hotbars?.length === 0) {
-      this.hotbars = [{
-        id: uuid.v4(),
-        name: "Default",
-        items: this.defaultHotbarInitialItems,
-      }];
-    } else {
```

Comment:
This was moved to the `alpha.0` migration so that it is always added. This code doesn't work when moving from pre-alpha.0 because the migration of `beta.10` adds at least one hotbar.